blog/scripts/scripts.js

```diff
@@ -118,7 +118,8 @@ const CONFIG = {
 // Load LCP image immediately
 (function loadLCPImage() {
   const lcpImg = document.querySelector('img');
-  lcpImg?.removeAttribute('loading');
+  lcpImg?.setAttribute('loading', 'eager');
+  lcpImg?.setAttribute('fetchpriority', 'high');
```

Comment:
FYI the [default is already eager.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading#value) 
https://github.com/adobecom/milo/pull/1908#discussion_r1498084211

The difference you are going to see with `fetchpriority` is if there are multiple eager loading images.
> [high](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority#high)
> Fetch the image at a high priority relative to other images.